source/agora/common/crypto/Key.d

```diff
@@ -213,3 +282,19 @@ public enum VersionByte : ubyte
     /// Base32-encodes to 'X...'
 	HashX = 23 << 3,
 }
+
+// Test with a stable keypair
+// We cannot actually test the content of the signature since it includes
+// random data, so it changes every time
```

Comment:
Ah, is this to prevent replay attacks? Is it documented somewhere in libsodium?

Reply:
No it's to prevent leaking the key.
Remember Schnorr signatures ? That random `r` parameter ? Same thing.

Reply:
Right, but I mean if you sign the same data twice you get a different signature? Can I read about this somewhere?

Reply:
https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Signature_generation_algorithm